include/ubiquity_motor/motor_parmeters.h

```diff
@@ -116,6 +119,8 @@ struct FirmwareParams {
             nh, "ubiquity_motor/fw_max_pwm", max_pwm);
         deadman_timer = getParamOrDefault(
             nh, "ubiquity_motor/deadman_timer", deadman_timer);
+        hw_options = getParamOrDefault(
```

Comment:
I strongly object to this being a parameter, it should *only* ever be read by the firmware

Reply:
Agreed.  Not sure why I had it as settable from ROS side but at the time I was thinking we may want to make it switchable in some way which we discussed. I had not gone back and removed these 2 lines after that.   It does seem that it is a sort of read only member of FirmwareParams and so I leave it there so it is tightly bound to firmware, that part seems right.  The option we had discussed on the phone was if we ever wanted to try to make it switchable and I am glad you agreed we do not as that would be problematic.